package.json

```diff
@@ -10,13 +10,14 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^5.2.1",
```

Comment:
### OPTIONAL COMMENT:
- [ ] It would be better if we capitalize our commit messages, this helps our repository looks professional